daemon/emer-aggregate-tally.c

```diff
@@ -178,16 +187,19 @@ delete_tally_entries (EmerAggregateTally  *self,
     }
   g_string_append (query, ");");
 
-  if (!CHECK (sqlite3_prepare_v2 (self->db, query->str, -1, &stmt, NULL)) ||
-      !CHECK (sqlite3_step (stmt)) ||
-      !CHECK (sqlite3_reset (stmt)) ||
```

Comment:
`sqlite3_reset()` followed by `sqlite3_finalize()` is redundant, which is why I have removed it.